lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -672,6 +672,8 @@ defmodule Kernel.SpecialForms do
   defmacro __CALLER__
 
   @doc """
+  Pin operator.
```

Comment:
yes, that's alright.

I changed the second mention to the pin operator to

> This can be done with the `^` special form,  colloquially known as the pin operator:

so it doesn't repeat the exact same phrase using "Also known as"